packages/devtools_app/lib/src/screens/profiler/cpu_profile_model.dart

```diff
@@ -701,6 +697,16 @@ class CpuStackFrame extends TreeNode<CpuStackFrame>
     ].join(' - ');
   }
 
+  /// [copySampleCounts] control whether or not the resulting [CpuStackFrame]
+  /// will have [exclusiveSampleCount] and [inclusiveSampleCount] initialized.
+  ///
+  /// Sample counts should only be reset when building a filtered view of the
+  /// full set of samples, as some stacks may no longer be included in the
+  /// profile, changing the exclusive counts.
+  ///
+  /// Inclusive sample counts are also reset by default, unless
```

Comment:
maybe add here: "When [copySampleCounts] is true, inclusive counts are reset by default..." to make it clear that the `resetInclusiveSampleCounts` param only does something if `copySampleCounts` is true.